experimental/iterators/test/python/dialects/iterators/dialect.py

```diff
@@ -1,11 +1,16 @@
 # RUN: %PYTHON %s | FileCheck %s
 
+import ctypes
 import os
 
+import pandas as pd
+import numpy as np
+
+from mlir_iterators.runtime.pandas_to_iterators import to_columnar_batch_descriptor
```

Comment:
On my machine, these kinds of imports seem to crash as it tries to find `runtime` in `mlir_iterators` of the python_bindings. Maybe rename?